src/css/main.less

```diff
@@ -1307,27 +1304,56 @@ fieldset[disabled] .btn-info.active {
       height: @base-spacing-unit*2;
       margin: 0 0 0 @base-spacing-unit;
       width: @base-spacing-unit*2;
+      vertical-align: baseline;
     }
 
     &:last-child {
-      &::after {
-        opacity: 0;
-      }
-
       a {
         color: darken(@link-color, 40%);
         cursor: default;
+
+        &::before {
+          content: "";
+        }
+      }
+
+      &::after {
+        opacity: 0;
       }
     }
 
     a {
       color: @link-color;
+      white-space: nowrap;
+      overflow: hidden;
 
       &:hover {
         color: darken(@link-color, 40%);
       }
     }
   }
+
+  &.collapsed {
+    a {
+      width: 0.75em;
+
+      &::before {
```

Comment:
Not sure.
The bluebars come from the first pixel of the "p" in "longgroupname". hehe